pyccel/ast/builtin_methods/set_methods.py

```diff
@@ -13,8 +13,7 @@
 from pyccel.ast.internals import PyccelInternalFunction
 from pyccel.ast.basic import TypedAstNode
 
-__all__ = ('SetAdd', 'SetClear', 'SetMethod', 'SetCopy', 'SetPop', 'SetRemove')
-
+__all__ = ('SetAdd', 'SetClear', 'SetMethod', 'SetCopy', 'SetPop', 'SetRemove', 'SetDiscard')
```

Comment:
There are starting to be a lot of these. Can we order them alphabetically and put each one in its own line like we do in most of the other files? This makes it easier to quickly determine if a specific Set method is supported

Reply:
Fixed : 55d1560

Reply:
> and put each one in its own line like we do in most of the other files?

This bit still seems to be missing

Reply:
Fixed :  9fd3260